src/android/data/LocationSerializer.java

```diff
@@ -0,0 +1,47 @@
+package com.tenforwardconsulting.cordova.bgloc.data;
+
+import com.tenforwardconsulting.cordova.bgloc.data.Location;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.json.JSONException;
+
+/**
+ * Helper class to translate Locations into JSON objects.  The JSON objects aim to mimick the https://developer.mozilla.org/en-US/docs/Web/API/Position API.
```

Comment:
I guess it's not a backwards compatibility breaking change because only new functionality is using this LocationSerializer.  However, I don't think the LocationSerializer as it stands should make it into the main codebase.  

I implemented it this way because we at Hurdlr happen to only be using this format.  I figured before fixing the LocationService class I'd get feedback from you :)